plugin-copilot/src/main/java/appland/copilotChat/copilot/CopilotContentExclusion.java

```diff
@@ -0,0 +1,75 @@
+package appland.copilotChat.copilot;
+
+import appland.utils.GsonUtils;
+import com.google.gson.annotations.SerializedName;
+import com.intellij.util.Url;
+import com.intellij.util.Urls;
+import com.intellij.util.io.HttpRequests;
+
+import javax.annotation.Nullable;
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+
+public record CopilotContentExclusion(@SerializedName("rules") List<Rule> rules,
+                                      @SerializedName("last_updated_at") String lastUpdatedAt,
+                                      @SerializedName("scope") Scope scope) {
+    /**
+     * Fetches the global exclusion rules from the Copilot API.
+     *
+     * @param gitHubToken The GitHub token to use for authentication.
+     * @return The content exclusion rules.
+     * @throws IOException
+     */
+    static public List<CopilotContentExclusion> fetchGlobal(String gitHubToken) throws IOException {
```

Comment:
nit: I usually add `@NonNull` to make it clear what's expected.
```suggestion
    static public List<CopilotContentExclusion> fetchGlobal(@NotNull String gitHubToken) throws IOException {
```